src/main/java/com/hivemq/configuration/service/entity/TcpListener.java

```diff
@@ -93,4 +90,49 @@ public String readableName() {
         return name;
     }
 
+    public static class Builder {
```

Comment:
You can add a static method `builder()` to the class which returns an instance of the Builder Class.

Reply:
The current hierarchy of the classes though doesn't allow static builder methods.